src/coreclr/pal/inc/mbusafecrt.h

```diff
@@ -58,11 +58,7 @@ extern errno_t strncpy_s( char* outDest, size_t inDestBufferSize, const char* in
 extern errno_t wcsncpy_s( WCHAR* outDest, size_t inDestBufferSize, const WCHAR* inSrc, size_t inCount );
 extern errno_t wcsncpy_s( WCHAR* outDest, size_t inDestBufferSize, const WCHAR* inSrc, size_t inCount );
 
-// strnlen is not required unless the source string is completely untrusted (e.g. anonymous input on a website)
-#ifndef SUPPRESS_STRNLEN
-    extern size_t PAL_strnlen( const char* inString, size_t inMaxSize );
-    extern size_t PAL_wcsnlen( const WCHAR* inString, size_t inMaxSize );
-#endif
+extern size_t PAL_wcsnlen( const WCHAR* inString, size_t inMaxSize );
```

Comment:
Can you move this one to the minipal where the other u16 APIs are being located?

Reply:
I have another branch where I'm moving over the WCHAR-based APIs to the CoreCLR minipal. I'd rather delay this work to be part of that work.